discord/client.py

```diff
@@ -382,9 +382,9 @@ async def _run_event(
             await coro(*args, **kwargs)
         except asyncio.CancelledError:
             pass
-        except Exception:
+        except Exception as exc:
             try:
-                await self.on_error(event_name, *args, **kwargs)
+                await self.on_error(event_name, exc, *args)
```

Comment:
`kwargs` should still be passed
```suggestion
                await self.on_error(event_name, exc, *args, **kwargs)
```